src/app/code/community/Shopgate/Cloudapi/etc/api2.xml

```diff
@@ -238,6 +241,14 @@
                     <group_id>Group</group_id>
                     <confirmation>Is Confirmed</confirmation>
                     <disable_auto_group_change>Disable automatic group change based on VAT ID</disable_auto_group_change>
+                    <dob>Customer date of birth</dob>
```

Comment:
not sure it's needed as it's not in the `app/code/core/Mage/Customer/etc/api2.xml`, but who knows, I will double check when logic is adjusted